cachito/workers/requests.py

```diff
@@ -29,7 +33,12 @@ def get_requests_session(auth=False):
             session.cert = config.cachito_auth_cert
 
     retry = Retry(
-        total=5, read=5, connect=5, backoff_factor=1.3, status_forcelist=(500, 502, 503, 504)
+        total=5,
+        read=5,
+        connect=5,
+        backoff_factor=1.3,
+        status_forcelist=(500, 502, 503, 504),
```

Comment:
You said you wanted to remove 500 from the status list, do you still think that should be done?

Reply:
I don't sure for now. Because I don't have statistics about how often the 500 status is caused by internal errors and how often it is only an external outage (so it makes sense to repeat the request).

Reply:
That's a good point, let's keep it for now